src/contrib/image/image.cc

```diff
@@ -0,0 +1,105 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file Use external image library (jpeg, png, bml, gif ...etc) call.
+ */
+#include <tvm/runtime/registry.h>
+#include <tvm/runtime/util.h>
+#include <dmlc/logging.h>
+
+extern "C" {
+#include <jpeglib.h>
+#include <setjmp.h>
+
+struct jpeg_internal_source_mgr {
+    struct jpeg_source_mgr pub;
```

Comment:
Please fix the style. need to use two space indent

Reply:
Done